configuration/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "configuration"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1.0.70"
+aws-credential-types = "1.0.0"
+aws-sdk-s3 = { version = "0.39.1", features = ["behavior-version-latest"] }
+aws-types = "1.0.0"
+dotenv = "0.15.0"
+lazy_static = "1.4.0"
+project-root = "0.2.2"
+regex = "1.10.2"
+serde = "1.0.145"
+serde_derive = "1.0.145"
+serde_json = "1.0.108"
+opentelemetry = { version = "0.19", features = ["rt-tokio-current-thread"] }
+opentelemetry-jaeger = { version = "0.18", features = [
+    "rt-tokio-current-thread",
+    "collector_client",
+    "isahc_collector_client",
+], optional = true }
+toml = "0.8.4"
+tracing = "0.1.34"
+tracing-subscriber = { version = "0.3.15", features = [
+    "fmt",
+    "env-filter",
+    "std",
+    "json",
+] }
+tracing-opentelemetry = { version = "0.19", optional = true }
+tracing-stackdriver = "0.7.2" # GCP logs
+
+
+near-indexer-primitives = "1.36.0"
```

Comment:
Do you really need it explicitly? These primitives reexported from the `near-lake-framework`